script.js

```diff
@@ -1,4 +1,118 @@
+var win_count = 0;
+var loss_count = 0;
+var name_input = "";
+
+var random_pss = function () {
+  var paper = "Paper";
+  var scissors = "Scissors";
+  var stone = "Stone";
```

Comment:
Might be too elaborate, in lines 13/15/16 we can just return "scissors" / return "paper" / return "stone"

We don't always have to return variables - in simple cases like this we can simply return the values that we want as output!